test/junit/scala/lang/stringinterpol/StringContextTest.scala

```diff
@@ -173,7 +173,7 @@ class StringContextTest {
       f"${s}%S"     -> "SCALA",
       f"${5}"       -> "5",
       f"${i}"       -> "42",
-      f"${'foo}"    -> "'foo",
+      f"${'foo}"    -> "Symbol(foo)",
```

Comment:
Just to note that this will conflict with https://github.com/scala/scala/pull/8890/files#diff-bd815597e6f8f90ee8858d87776722ddR180 

I don't know if the merge is easier if you also amend the expression not to use symbol syntax?

Reply:
It's better not to use symbol syntax, but it will still conflict with #8890 on the same line.

Reply:
I guess you're right. No problem, though!